frontend/components/Home/Dashboard/Dashboard.tsx

```diff
@@ -62,6 +60,40 @@ const Dashboard = () => {
     const [logOpen, setLogOpen] = useState(false);
     const [logModal, setLogModal] = useState(false);
 
+    const [messageDisp, setMessageDisp] = useState(false);
+   
+    const [tipCard, setTipCard] = useState({title: "", description: "", student: false});
+
+    var tipsArr = tips;
+
+    useEffect(() => {
+        
+        //check membership
+        if(!canCreateCourse) {
+            //student
+            tipsArr = tips.filter( x => x.student == true );
+            const currentDate = new Date();
+            const index = (currentDate.getDate())  % tipsArr.length;
+            setTipCard(tipsArr[index]);
+        } else {
+            //instructor
+            const currentDate = new Date();
+            const index = (currentDate.getDate())  % tips.length;
+            setTipCard(tips[index]);
+        }
+
+        const currentDate = new Date();
+        const today = currentDate.toDateString();
+        
+        console.log(today);
+        console.log(localStorage.getItem("lastSeenDate"));
+        if (!(today === localStorage.getItem("lastSeenDate"))) {
```

Comment:
This conditional can be simplified down to just setMessageDisp(today !== localStorage.getItem("lastSeenDate");